grammars/perl.cson

```diff
@@ -350,9 +350,9 @@
   }
   {
     'applyEndPatternLast': 1
-    'begin': '\\b(?=(?<!\\&)(s)(\\s+\\S|\\s*[;\\,\\#\\{\\}\\(\\)\\[<]|$))'
```

Comment:
yea, this part here is for regex matches with "start" and "end" tags like: 
```
s<asd><def>
s(ddd)(sss)
```
but "#" is not a "tag" so its just
```
s#asd#def#
```